packages/@uppy/companion/src/server/controllers/thumbnail.js

```diff
@@ -11,8 +11,8 @@ async function thumbnail (req, res, next) {
   const { accessToken } = providerUserSession
 
   try {
-    const { stream } = await provider.thumbnail({ id, token: accessToken, providerUserSession })
-    res.set('Content-Type', 'image/jpeg')
+    const { stream, contentType = 'image/jpeg' } = await provider.thumbnail({ id, token: accessToken, providerUserSession })
+    res.set('Content-Type', contentType)
```

Comment:
Not sure, but shouldn't the default be empty and the header not sent? This might still break for custom providers not returning jpeg streams..

Reply:
Not sure also. Before this there wasn’t any content type at all. Is a wrong content type worse than a missing content type? I don’t know

Reply:
I *assume*, clients try to figure out a type unless it's set. If it's set, they should interpret it as such, so mismatch sounds worse to me than a missing type... of course I don't have any spec at hand to substantiate 😬